### PR TITLE
[FLOC-2607] Fix invalid Eliot log message from EBS driver.

### PR DESCRIPTION
**Pull request by myechuri**
Fix bug in EBS driver that causes failure in logging devices in use while attempting to find an available device name during attach_volume.

### CODE REVIEW

File: flocker/node/agents/functional/test_ebs.py [hunk 1]
```diff
@@ -163,6 +164,31 @@ def test_next_device_in_use(self):
...
+    @capture_logging(
+        assertHasMessage, IN_USE_DEVICES, {
+            'devices': [u'/dev/sda1', u'/dev/sdf']
+        },
+    )
+    def test_in_use_devices_log(self, logger):
+        """
+        Test that attached device shows up as being in use during subsequent
+        ``attach_volume``.
+        """
```
Comment (jml) [inline flocker/node/agents/functional/test_ebs.py]: Nit: You never need to say "Test that" in a test docstring. You can just described the behaviour, e.g. "The attached device shows up ..."

https://jml.io/test-docstrings/

**Comment by jml** [comment]
:+1: Thanks! Minor change requested. Please merge after fixing it, and when the tests pass.

File: flocker/node/agents/functional/test_ebs.py (hunk 1 above)
Comment (myechuri) [inline flocker/node/agents/functional/test_ebs.py]: Thanks! Fixed in other places as well.

**Comment by myechuri** [comment]
Tests passed with the latest changes, going ahead with merge. Thanks, @jml , for the review!

**Pull request merged**